contribs/gmf/apps/iframe_api/Controller.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @module app.iframe_api.Controller
+ */
+/**
+ * Application entry point.
+ *
+ * This file includes `goog.require`'s for all the components/directives used
+ * by the HTML page and the controller to provide the configuration.
+ */
+
+import './sass/iframe_api.scss';
+import gmfControllersAbstractDesktopController from 'gmf/controllers/AbstractDesktopController.js';
+import appBase from '../appmodule.js';
+import ngeoProjEPSG2056 from 'ngeo/proj/EPSG2056.js';
+import ngeoProjEPSG21781 from 'ngeo/proj/EPSG21781.js';
+import * as olBase from 'ol/index.js';
+import olAttribution from 'ol/control/Attribution.js';
+import Raven from 'raven-js/src/raven.js';
+import RavenPluginsAngular from 'raven-js/plugins/angular.js';
+
+if (!window.requestAnimationFrame) {
+  alert('Your browser is not supported, please update it or use another one. You will be redirected.\n\n'
+    + 'Votre navigateur n\'est pas supporté, veuillez le mettre à jour ou en utiliser un autre. Vous allez être redirigé.\n\n'
+    + 'Ihr Browser wird nicht unterstützt, bitte aktualisieren Sie ihn oder verwenden Sie einen anderen. Sie werden weitergeleitet.');
+  window.location = 'http://geomapfish.org/';
+}
+
+/**
+ * @param {angular.Scope} $scope Scope.
+ * @param {angular.$injector} $injector Main injector.
+ * @constructor
+ * @extends {gmf.controllers.AbstractDesktopController}
+ * @ngInject
+ * @export
+ */
+const exports = function($scope, $injector) {
+  gmfControllersAbstractDesktopController.call(this, {
```

Comment:
I it a good idea to use the AbstractDesktopController?
For me he contains many things that not useed on API interface ...

Reply:
That's a good point.  Indeed, AbstractDesktopController contains more things that are not required by IframeAPI those required:

 * map creation
 * feature helper / vector features (if we want to load vector features in the map)
 * isDesktop true
 * query options

If you think it is best to have those copied in the controller of the app and extend AbstractAppController, please let me know.

Reply:
Isn't the best to create an AbstractAPIController and AbstractDesktopController will extend it?

Reply:
Sure, I'll do that.